geomesa-convert/geomesa-convert-common/src/main/scala/org/locationtech/geomesa/convert2/transforms/StringFunctionFactory.scala

```diff
@@ -8,21 +8,54 @@
 
 package org.locationtech.geomesa.convert2.transforms
 
+import org.apache.commons.lang.StringUtils
+
 import scala.util.matching.Regex
 
 
 class StringFunctionFactory extends TransformerFunctionFactory {
 
   override def functions: Seq[TransformerFunction] =
-    Seq(stripQuotes, strLen, trim, capitalize, lowercase, uppercase, regexReplace, concat,
+    Seq(stripQuotes, strip, stripPrefix, stripSuffix, replace, removeChars,
+      strLen, trim, capitalize, lowercase, uppercase, regexReplace, concat,
       substr, string, mkstring, emptyToNull, printf)
 
   private val string = TransformerFunction("toString") { args =>
     args(0).toString
   }
 
   private val stripQuotes = TransformerFunction("stripQuotes") { args =>
-    args(0).asInstanceOf[String].replaceAll("\"", "")
+    StringUtils.strip(args(0).asInstanceOf[String], "'\"")
+  }
+
+  private val strip = TransformerFunction("strip") { args =>
+    if (args.length == 1) {
+      StringUtils.strip(args(0).asInstanceOf[String])
+    } else {
+      val toStrip = args(1).asInstanceOf[String]
+      StringUtils.strip(args(0).asInstanceOf[String], toStrip)
+    }
+  }
+
+  private val stripPrefix = TransformerFunction("stripPrefix") { args =>
+    val toStrip = args(1).asInstanceOf[String]
+    StringUtils.stripStart(args(0).asInstanceOf[String], toStrip)
+  }
+
+  private val stripSuffix = TransformerFunction("stripSuffix") { args =>
+    val toStrip = args(1).asInstanceOf[String]
+    StringUtils.stripEnd(args(0).asInstanceOf[String], toStrip)
```

Comment:
As an idea, are there other methods from StringUtils that we'd like to go ahead and surface as converters?

@elahrvivaz thoughts?

Reply:
i'm sure there are a few...i will try to do a review later on.